Tools/patchcheck/untabify.py

```diff
@@ -49,7 +49,8 @@ def process(filename, tabsize, verbose=True):
         f.write(newtext)
     if verbose:
         print(filename)
+    return 1
 
 
 if __name__ == '__main__':
-    main()
+    raise SystemExit(main())
```

Comment:
`process()` now has return values, which go to `main()`, but `main()` always returns `None` so we never get any errors here.

Reply:
Oh, I didn't realise! ``pre-commit`` was failing in my local testing when I introduced tabs, perhaps it just fails when a file is changed after the hook has run.